src/localization.js

```diff
@@ -201,7 +201,7 @@ angular.module('ngLocalize')
         }
 
         function applySubstitutions(text, subs) {
-            var res = text,
+            var res = (angular.isArray(text) ? text.join('') : text),
```

Comment:
Based on the use case presented for the split if we want to split "this is a very long string to be splitted in two" we can split in [ "this is a very long string ","to be splitted in two" ] and after the join the string will be "this is a very long string to be splitted in two" if the join is an space it will become "this is a very long string  to be splitted in two" with a double space between "string" and "to"